### PR TITLE
Reaction page

### PR DESCRIPTION
**Pull request by 0xPratik**

### CODE REVIEW

**Comment by 0xPratik** [comment]
Screenshot 2023-08-22 at 2 10 38 PM

File: src/view/screens/MissionsTab.tsx [hunk 1]
```diff
@@ -91,46 +119,53 @@ const DisplayReactions = observer(() => {
...
-            }}
-          />
-        </View>
-      </TouchableOpacity>
+          /> */}
+        <FlatList
+          data={GENESIS_REACTIONS}
+          numColumns={isMobileWeb ? 4 : 4}
+          key={4}
+          renderItem={({ item }) => {
```
Comment (tj-dispatch) [inline src/view/screens/MissionsTab.tsx:140]: move this into its own component please

File: src/view/screens/MissionsTab.tsx [hunk 2]
```diff
@@ -180,23 +217,41 @@ export const MissionsTab = withAuthRequired(
...
-              <Link href="/profile/viksit.live.solarplex.xyz">
-                @viksit.live.solarplex.xyz
-              </Link>{" "}
-            </Text>
-          )}
-          <DisplayReactions />
-        </ScrollView>
-      </CenteredView>
+              {!store.reactions.earnedReactions["genesis"]?.length && (
+                <Text type="lg-heavy" style={[pal.text, styles.textPadding]}>
```
Comment (tj-dispatch) [inline src/view/screens/MissionsTab.tsx:221]: own component

File: src/view/screens/MissionsTab.tsx [hunk 3]
```diff
@@ -62,27 +87,30 @@ const DisplayReactions = observer(() => {
...
       ]}
     >
-      <TouchableOpacity onPress={() => onPressReactionPack("genesis")}>
-        <View style={styles.HeaderRow}>
-          <View style={styles.horizontalView}>
-            <UserAvatar
-              size={40}
-              avatar={"https://i.ibb.co/NLkvySY/blob.png"}
-            />
+      <View style={s.p10}>
```
Comment (tj-dispatch) [inline src/view/screens/MissionsTab.tsx:98]: own component

File: src/view/screens/MissionsTab.tsx [hunk 4]
```diff
@@ -62,27 +87,30 @@ const DisplayReactions = observer(() => {
...
+        <InfoText text="Reaction packs are on chain collectibles that allow you to uniquely express yourself on Solarplex posts. Engage with/create posts to win points and unlock packs!" />
+      </View>
+
+      <View style={styles.HeaderRow}>
+        <View style={styles.horizontalView}>
+          <UserAvatar size={25} avatar={"https://i.ibb.co/NLkvySY/blob.png"} />
 
+          <View style={styles.verticalView}>
```
Comment (tj-dispatch) [inline src/view/screens/MissionsTab.tsx:106]: own component

**Review by tj-dispatch — CHANGES_REQUESTED** [review]

**Pull request merged**